api/endpoints/slack-auth.js

```diff
@@ -11,12 +11,13 @@ export default async (req, res) => {
 
   console.log({code, recordIDData, userID, meetingID})
 
+  const tokenRedirectUri = process.env.NODE_ENV === "production" ? "https://hack.af/z/slack-auth" : "https://slash-z-staging-1ae8b1c9e24a.herokuapp.com/api/endpoints/slack-auth";
```

Comment:
Do we need to update the manifest.yml for this change?

Reply:
don't think so
I assume this was kept here in case we need to recreate the bot with same permissions (we can copy-paste into Slack when creating a bot iirc)